src/transaction/hooks/useWriteContracts.ts

```diff
@@ -8,9 +8,10 @@ type UseWriteContractsParams = {
   setTransactionId: (id: string) => void;
 };
 
-const genericErrorMessage = 'Something went wrong. Please try again.';
+export const genericErrorMessage = 'Something went wrong. Please try again.';
 const uncaughtErrorCode = 'UNCAUGHT_WRITE_TRANSACTIONS_ERROR';
 const errorCode = 'WRITE_TRANSACTIONS_ERROR';
+const eoaErrorMesssage = 'this request method is not supported';
```

Comment:
thoughts on combining all errors from all components into a single file or module called `errors`

Reply:
+ I think this would be great, especially as we align on a standardized error approach - that refactor would be better suited in a future PR